elastic/shared/runners/datastream.py

```diff
@@ -50,6 +50,33 @@ async def create(es, params):
     return ops, "ops"
 
 
+class DeleteRemoteDataStream(Runner):
+    multi_cluster = True
+
+    def __init__(self):
+        super().__init__()
+
+    async def __call__(self, multi_es, params):
+        """
+        Deletes specified datastreams from all clusters
+        """
+        ops = 0
+        for cluster_name, cluster_client in multi_es.items():
+            data_stream = mandatory(params, "data-stream", "delete-datastream")
+            logger = logging.getLogger(__name__)
```

Comment:
Just to keep it lean, can we move this to an instance variable in `__init__`?

Reply:
actually, if you do that, there's no longer a reason to import from `esrally.driver.runner` and we can instead duck type this runner class

Reply:
double-actually, I now see the `mandatory` function invocation you're borrowing from `Runner`. I still think duck-typing is worthwhile (and therefore not using `mandatory`) since we only use this runner here, and wouldn't anticipate a scenario where `data-stream` isn't defined anyway

Reply:
Addressed in https://github.com/elastic/rally-tracks/pull/333/commits/73c2bf289d88ddb3fa35c7f092dd7cffdbf59fc8